Include/internal/pycore_pyerrors.h

```diff
@@ -130,6 +130,18 @@ PyAPI_FUNC(void) _PyErr_SetString(
     PyObject *exception,
     const char *string);
 
+/*
+ * Set an exception with the error message decoded from the current locale
+ * encoding (LC_CTYPE).
+ *
+ * Exceptions occurring in decoding take priority over the desired exception,
+ * in which case, this returns -1. Otherwise this returns 0 if the localized
+ * exception has been successfully set.
```

Comment:
```suggestion
 * Exceptions occurring in decoding take priority over the desired exception.
 * In those cases, this function returns -1. Otherwise this returns 0 if the
 * localized exception has been successfully set.
```